### PR TITLE
33 system requirements

### PR DESCRIPTION
**Pull request by nsolov**
specify required IRIS version
and production enabled namespace for a module

### CODE REVIEW

File: src/cls/_ZPM/PackageManager/Developer/SystemRequirements.cls [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+	Set tSystemMinor = $System.Version.GetMinor()
+	If (..VersionString = "") { Return $$$OK } 
+	try {
+		Set tOperator = ..Version.Options.GetAt(1).Comparators.GetAt(1).Operator
+		Set tModuleMajor = ..Version.Options.GetAt(1).Comparators.GetAt(1).Major
+		Set tModuleMinor = ..Version.Options.GetAt(1).Comparators.GetAt(1).Minor
+		Set tVersionMatch = 0
+		Set tCommand = "(tSystemMajor,tModuleMajor,tSystemMinor,tModuleMinor,tVersionMatch) If ((tSystemMajor "_tOperator_" tModuleMajor) && (tSystemMinor "_tOperator_" tModuleMinor)) { Set tVersionMatch = 1 }"
+
+		Xecute (tCommand, tSystemMajor,tModuleMajor,tSystemMinor,tModuleMinor,.tVersionMatch)
```
Comment (daimor) [inline src/cls/_ZPM/PackageManager/Developer/SystemRequirements.cls:41]: Xecute is good command in some cases, but not here. Should be replaced with $case, with a fixed list of operators.

File: src/cls/_ZPM/PackageManager/Developer/SystemRequirements.cls [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+{
+	Set i%VersionString = tValue
+	Set tSC = ##class(%ZPM.PackageManager.Core.SemanticVersionExpression).FromString(tValue,.tExpression)
+	If $$$ISOK(tSC) {
+		Set ..Version = tExpression
+	}
+	Quit tSC
+}
+
+Property Product As %String(XMLPROJECTION = "ATTRIBUTE");
```
Comment (daimor) [inline src/cls/_ZPM/PackageManager/Developer/SystemRequirements.cls:18]: not used

File: src/cls/_ZPM/PackageManager/Developer/SystemRequirements.cls (hunk 1 above)
Comment (nsolov) [reply src/cls/_ZPM/PackageManager/Developer/SystemRequirements.cls]: I changed to leverage existing version evaluation code, without Xecute.

We discussed GetFeature with Evgeny and he  said that it's not necessary now

File: src/cls/_ZPM/PackageManager/Developer/SystemRequirements.cls (hunk 2 above)
Comment (nsolov) [reply src/cls/_ZPM/PackageManager/Developer/SystemRequirements.cls]: removed

**Comment by daimor** [comment]
Could you check what's going on in tests, why they failed?

**Pull request merged**